### PR TITLE
Add community governance section to contributing.md

### PR DESCRIPTION
**Pull request by kylebarron**
Copied from the deck.gl `contributing.md`. I took a guess as to the TSC of loaders.gl

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ Contributions are welcome, assuming that they align with the general design goal
...
 
+## Community Governance
+
+vis.gl is part of the [Urban Computing Foundation](https://uc.foundation/). See the organization's [Technical Charter](https://github.com/visgl/tsc/blob/master/Technical%20Charter.md).
+
+### Technical Steering Committee
+
+loaders.gl development is governed by the vis.gl Technical Steering Committee (TSC). Current members:
+
+- [Ib Green](https://github.com/ibgreen)
```
Comment (ibgreen) [inline docs/contributing.md:15]: I don't think we should duplicate the member list of the TSC here, that sounds like a maintenance trap. Rather link to that doc in the tsc repo. 

And focus on listing the key maintainers for this repo. We can move these names to the next section.

**Comment by ibgreen** [comment]
Reading again in the charter

```
### 2c. Becoming a maintainer

Any contributor to the project can self-nominate to become a maintainer if they:

- Demonstrate familiarity with the project and expertise in the relevant domain (e.g. WebGL, 3D tiles, Google Maps, React, etc.):
  + Obtain the endorsement by an existing maintainer, or
  + Have submitted at least one substantial PR (a complete feature or major bug fix) and remained active for more than 3 months in online discussions and relevant projects
- Agree to the continuous maintenance of the self-declared feature area(s).

The nomination is formally submitted as a PR to update the contributing guide (`/CONTRIBUTING.md`), with self-declared ownership of one or more feature areas. Nominations are approved by the steering committee.
```

**Comment by ibgreen** [comment]
Based on the above, the maintainers should have "self-declare" ownership of some areas when applying. I suppose it would be nice to list areas in the table.

- In your case at minimum self-declared ownership of the "terrain module".
- @xintongxia is "3D tiles and I3S"
- In my case "overall architecture"

File: docs/contributing.md (hunk 1 above)
Comment (kylebarron) [reply docs/contributing.md:15]: Ok. The TSC member list is still in the deck.gl `Contributing.md`

**Comment by Pessimistress** [comment]
@kylebarron There is a CONTRIBUTING.md template for vis.gl projects. I just invited you to the template repo. Closing this in favor of #751

**Comment by kylebarron** [comment]
*In favor of #753

**Pull request closed without merging**